modules/Components/src/client/BasicFileValidateAndSave.coffee

```diff
@@ -174,6 +175,7 @@ class BasicFileValidateAndSaveController extends Backbone.View
 		@notificationController.addNotifications(@errorOwnerName, json.errorMessages)
 		if json.results?.htmlSummary?
 			@$('.bv_htmlSummary').html(json.results.htmlSummary)
+			@$('.bv_htmlSummary .bv_doseResponseSummaryTable').dataTable()
```

Comment:
Will this line be unhappy if there is no `.bv_doseResponseSummaryTable`?

Reply:
No, this line was ok but admittedly I hadn't checked before this.  I just added a safety check anyway.

Reply:
Made this just render the plain html table if it finds one and also added a warning so the user knows it couldn't be rendered as an interactive table.